tests/system/action/poll/test_stop.py

```diff
@@ -31,10 +35,79 @@ def test_stop_correct(self) -> None:
         self.assert_status_code(response, 200)
         poll = self.get_model("poll/1")
         assert poll.get("state") == "finished"
+        assert poll.get("votescast") == "1.000000"
+        assert poll.get("votesinvalid") == "0.000000"
+        assert poll.get("votesvalid") == "1.000000"
         countdown = self.get_model("projector_countdown/1")
         assert countdown.get("running") is False
         assert countdown.get("countdown_time") == 60
 
+    def test_stop_auto_calc_fields(self) -> None:
```

Comment:
Please remove the countdown from this test, it does not belong here

Reply:
Removed